dashboard/src/components/AppView/AppView.v2.tsx

```diff
@@ -0,0 +1,256 @@
+import { ClarityIcons, rewindIcon, trashIcon, uploadCloudIcon } from "@clr/core/icon-shapes";
+import { RouterAction } from "connected-react-router";
+import { assignWith } from "lodash";
+import { get } from "lodash";
+import React, { useEffect, useState } from "react";
+import * as yaml from "yaml";
+import placeholder from "../../placeholder.png";
+import { CdsButton, CdsIcon } from "../Clarity/clarity";
+
+import Alert from "components/js/Alert";
+import Column from "components/js/Column";
+import Row from "components/js/Row";
+import PageHeader from "components/PageHeader/PageHeader.v2";
+import { Link } from "react-router-dom";
+import ApplicationStatus from "../../containers/ApplicationStatusContainer";
+import helmIcon from "../../icons/helm.svg";
+import ResourceRef from "../../shared/ResourceRef";
+import { IK8sList, IRelease, IResource } from "../../shared/types";
+import * as url from "../../shared/url";
+import LoadingWrapper from "../LoadingWrapper/LoadingWrapper.v2";
+import AccessURLTable from "./AccessURLTable/AccessURLTable.v2";
+import AppNotes from "./AppNotes.v2";
+import AppSecrets from "./AppSecrets";
+import AppValues from "./AppValues/AppValues.v2";
+import "./AppView.v2.css";
+import ChartInfo from "./ChartInfo/ChartInfo.v2";
+import ResourceTabs from "./ResourceTabs";
```

Comment:
Woah - certainly pulling a lot of components together... that's a huge number of imports!

Reply:
Everything is a component hehe!